kebechet/managers/update.py

```diff
@@ -115,10 +115,14 @@ def _get_direct_dependencies() -> tuple:
         return default, develop
 
     @staticmethod
-    def _get_direct_dependencies_requirements() -> set:
-        """Get all direct dependencies based on requirements.in file and generated Pipfile.lock from it."""
-        with open('requirements.in') as requirements_in_file:
-            content = requirements_in_file.read()
+    def _get_direct_dependencies_requirements(req_dev: bool) -> set:
+        """Get all direct dependencies based on either requirements.in or requirements-dev.in file and generated Pipfile.lock from it."""
+        if req_dev:
+            with open('requirements-dev.in') as requirements_in_file:
```

Comment:
Try to reduce this by assigning the input file into a variable:
```python
input_file = 'requirements-dev.in' if req_dev else 'requirements.in'
with open(input_file, 'r') as requirements_file:
        content = requirements_file.read()
```

Here and other places bellow.